pkg/stacker/build.go

```diff
@@ -706,14 +729,41 @@ func SetupLayerConfig(config types.StackerConfig, c *container.Container, l type
 	importsDir := path.Join(config.StackerDir, "imports", name)
 	if _, err := os.Stat(importsDir); err == nil {
 		log.Debugf("bind mounting %s into container", importsDir)
-		err = c.BindMount(importsDir, "/stacker", "ro")
+		err = c.BindMount(importsDir, "/stacker/imports", "ro")
 		if err != nil {
 			return err
 		}
 	} else {
 		log.Debugf("not bind mounting %s into container", importsDir)
 	}
 
+	// make the artifacts path available so that boms can be built from the run directive
+	if l.Bom != nil {
+		artifactsDir := path.Join(config.StackerDir, "artifacts", name)
+		if _, err := os.Stat(artifactsDir); err == nil {
+			log.Debugf("bind mounting %s into container", artifactsDir)
+			err = c.BindMount(artifactsDir, "/stacker/artifacts", "rw")
+			if err != nil {
+				return err
+			}
+		} else {
+			log.Debugf("not bind mounting %s into container", artifactsDir)
+		}
+
+		// make stacker also available to run the internal bom cmds
+		binary, err := os.Readlink("/proc/self/exe")
+		if err != nil {
+			return errors.Wrapf(err, "couldn't find executable for bind mount")
+		}
+
+		err = c.BindMount(binary, "/stacker/tools/static-stacker", "")
```

Comment:
Is there a reason to call this 'static-stacker' rather than just 'stacker' ?

Reply:
Just keeping with existing convention and also run-on /stacker/tools/stacker?

Reply:
i think that makes sense.